### DISCUSSION TITLE
Error while executing gcode

### DISCUSSION THREAD

**Discussion by Muelio** [winder/Universal-G-Code-Sender#2032] [Help] [answered]
I have been working with ugs for a while now and today i came upon a new bug, when i am running an operation from fusion360 as i usually do it stops mid work at random locations, either with an error report or just a "o" or a "k" instead of "ok" after the last comand. Any idea what is causing it and how to fix?
![image](https://user-images.githubusercontent.com/79712168/199277836-a5fcc663-7346-4d58-ad8a-8f07c7dfb893.png)

> X122.448Y-43.285Z-1.49F3000
[Error] Error while processing response <<Run|Pos:-538.822,-596.389,-57.231|Bf:0,128|F:2018>>

![image](https://user-images.githubusercontent.com/79712168/199278249-cad4eb36-198b-488e-93b5-acf5f3510bed.png)

**Comment by breiler** [comment, marked as answer]
When characters are being dropped like that in the communication it is often because of communication failure due to a bad USB-cable or a broken (or too cheap) Arduino.

**Reply by Muelio** [reply]
yeah i just now tried a new usb cable and it seems to be working now

**Reply by Muelio** [reply]
the arduino is a legit arduino uno, and it is brand new, it should not be the origin of the problem, bit after switching the usb cable for now at least it seems to be working again. thanks for the help
